README.md

```diff
@@ -259,18 +224,26 @@ environment variable. So set the variable before running the tests:
 
 
 ```
-cd v8
-export ZOSLIB_LIBPATH=`pwd`/third_party/zoslib/install/lib
+$ cd $HOME/zopen/dev/v8base/v8
+$ export ZOSLIB_LIBPATH=`pwd`/third_party/zoslib/install/lib
 ```
-TODO: resume here
 
 
+##### Build V8
 
 
-# Build V8
 ```
-cd v8
-export PATH=`pwd`/buildtools/zos:$PATH # for the 'ar' wrapper
-gn -v gen out/zos_s390x.release --args="is_debug=false treat_warnings_as_errors=false"
-V=1 ninja -v -j 12 -C out/zos_s390x.release/
+$ cd $HOME/zopen/dev/v8base/v8
+$ export PATH=`pwd`/buildtools/zos:$PATH # for the 'ar' wrapper
+$ gn -v gen out/zos_s390x.release --args="is_debug=false treat_warnings_as_errors=false" V=1 ninja -v -j 12 -C out/zos_s390x.release/
 ```
+
+
+output:
+
+```
+$ gn -v gen out/zos_s390x.release --args="is_debug=false treat_warnings_as_errors=false" V=1 ninja -v -j 12 -C out/zos_s390x.release/
+gn.py: Could not find gn executable at: /z/jd895801/zopen/dev/v8base/v8/buildtools/zos/gn
```

Comment:
Can you add a step in the setup to create a symbolic link for gn, e.g.
```
$ cd $HOME/zopen/dev/v8base/v8/buildtools/zos
$ ln -s <path-to-gn-port>/out/gn gn
```

Reply:
yes, let me do that.  Many thanks!